alembic/versions/5002e75c0604_add_task_parent_to_retain_history_after_.py

```diff
@@ -0,0 +1,21 @@
+"""Add task parent to retain history after split
+
+Revision ID: 5002e75c0604
+Revises: 33e34b3beaa3
+Create Date: 2016-05-10 22:37:31.383527
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '5002e75c0604'
+down_revision = '33e34b3beaa3'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('task', sa.Column('parent_id', sa.Integer(), nullable=True))
+
+def downgrade():
```

Comment:
In addition to this, I think it would be fairly easy to update the data for the already existing project. Guessing the parent task id by comparing the x, y for tiles with a different zoom from the one of the project is almost straight forward in my opinion. I can give a hand on this if you wish.